src/util.rs

```diff
@@ -350,3 +357,51 @@ pub fn process_new(
         procfs::process::Process::new(pid)
     }
 }
+
+
+
+#[cfg(target_os = "macos")]
+pub fn arch_from_pid(pid: pid_t) -> &'static str {
```

Comment:
`util.rs` is for function used in many columns, so I think it is better that `arch_from_pid` is placed in `arch.rs`.